src/model/model_helper.py

```diff
@@ -27,6 +28,18 @@ def is_empty(value):
     return (not value) and (value != 0) and (value != False)
 
 
+def update_output_files_with_vars(output_files, arguments):
+    for i, output_file in enumerate(output_files):
+        regex = "\$\$([a-zA-Z0-9-_]+)"  # Syntaxe is : $$my_variable (with two '$')
```

Comment:
I would suggest to change internal loop a bit: we can iterate over all arguments and build regex based on it:
e.g.
```
for i, output_file in enumerate(output_files):
  for argument in arguments:
    output_file = re.sub('\$\$' + argument, arguments[argument], output_file)
```
Also secure arguments should be excluded

One more thing/question. $$ syntax is already used for environment variables in couple of places. What do you think, won't reusing the syntax cause any confusion? As for me, it shouldn't be a problem

Reply:
+1 about the loop.

About the syntaxe, i didn't know it was used for environment variables.
To avoid bad surprise, it would be better if we use 2 differents syntaxes. So, since the "$$" syntaxe is already used, we could use "$$$" or "$_"? What do you think about? "$$$" is not use in contrary to "$_" which is used by PHP.